src/SiteLink.php

```diff
@@ -18,10 +18,18 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Michał Łazowik
+ * @author Thiemo Mättig
```

Comment:
"This change is rather unrelated no?"

-> No it is not, what makes you think it is?

> "Are you adding your name as author because you are adding some docs for private code?"

-> No, because I made changes to the code in the class before.

I asked the question because I found what I saw very strange. Which Is why I did not say "-1. this change is unrelated". If you had simply answered my question, we'd have saved us a lot of time.